jest.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
      */
     {
       ...sharedConfig,
-      modulePathIgnorePatterns: ["v2"],
+      displayName: "v1",
```

Comment:
Added `displayName` so we can get a nice annotation which _project_ a test was running under. I name this "v1" to just a get a nice alignment with "v2" but I'm struggling to come up with a better name. Any ideas?

![Screen Shot 2020-06-12 at 9 08 36 AM](https://user-images.githubusercontent.com/796573/84506018-af059900-ac8c-11ea-8a64-f761bd6fcb7c.png)

Reply:
Oh nice! I didn't know about display name 👍